main.cpp

```diff
@@ -886,6 +887,9 @@ static void print_help_msl()
 	                "\t\t<format> can be 'any32', 'any16', 'u16', 'u8', or 'other', to indicate a 32-bit opaque value, 16-bit opaque value, 16-bit unsigned integer, 8-bit unsigned integer, "
 	                "or other-typed variable. <size> is the vector length of the variable, which must be greater than or equal to that declared in the shader.\n"
 	                "\t\tUseful if shader stage interfaces don't match up, as pipeline creation might otherwise fail.\n"
+					"\t[--msl-raw-buffer-tess-input]:\n\t\tUse raw buffers for tessellation evaluation input.\n"
```

Comment:
Would it be better to call this `msl-raw-buffer-tese-input`? This option only applies to TESE, and afaik, the multi-patch-workgroup is the equivalent for TESC.

Reply:
Done.